lib/string_poster.rb

```diff
@@ -0,0 +1,7 @@
+class StringPoster
+
+  def self.to_stream(str)
+   	uri = URI.parse("http://localhost:8989/")
+	 	Net::HTTP.post_form(uri, "q" => str)
```

Comment:
Also it makes sense to extract `"http://localhost:8989/"` to constant for now. Later it can be configurable.